__tests__/app.test.js

```diff
@@ -183,4 +183,53 @@ describe('GET: /api/reviews', () => {
             })
     })
 
+})
+
+describe('GET: /api/reviews/:review_id/comments', () => {
+    it('responds with correct status code 200', () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .expect(200)
+    })
+    it('gives an array of objects with the following properties', () => {
+        return request(app)
+            .get('/api/reviews/2/comments')
+            .then((res) => {
+                const body = res.body.comments
+                expect(body).toBeInstanceOf(Object)
+                const commentProperties = {
+                    comment_id: expect.any(Number),
+                    review_id: expect.any(Number),
+                    body: expect.any(String),
+                    votes: expect.any(Number),
+                    author: expect.any(String),
+                    created_at: expect.any(String)
+                };
+                body.forEach(comment => {
+                    expect(comment).toMatchObject(commentProperties);
+                });
+            })
+    })
```

Comment:
I'd like to see another test here for a - Status 200, valid ID, but has no comments and responds with an empty array of comments 👍